earth_enterprise/integration_test/specs/genewimageryresource.spec

```diff
@@ -0,0 +1,62 @@
+Copyright 2019 Thermopylae Sciences and Technology
```

Comment:
I don't think we can claim Copyright on this. Let's either remove it or make it `Copyright 2019 Open GEE Contributors`